### DISCUSSION TITLE
Tokens don't show up on Tokens Tab

### DISCUSSION THREAD

**Discussion by lukasz-szewczyk-xtb** [six7/figma-tokens#378] [Q&A]
I've problem with adding tokens, when I making new color token named "button.primary.background.disabled" it isn't show up at Tokens tab, but it's appears in JSON:

 "button": {
    "primary": {
      "background": {
        "value": "$base.background.inverted",
        "type": "color",
        "disabled": {
          "value": "$base.background.tertiary",
          "type": "color"
        }
      },

It's makes token unable to connect with object. Can you help me with it?

**Comment by six7** [comment]
It seems you defined two tokens in one. Remove the disabled token then you should be good to go!
